man/ctags.1.rst.in

```diff
@@ -1739,34 +1752,34 @@ TMPDIR
 FILES
 -----
 
-/ctags.cnf (on MSDOS, MSWindows only)
-
-/etc/ctags.conf
 
-@prefix@/etc/ctags.conf
+$HOME/.ctags.d/\*.ctags
 
-$HOME/.ctags
+$HOMEDRIVE$HOMEPATH/ctags.d/\*.ctags
```

Comment:
Isn't it better to add " (on MSWindows only)" at the end of the line?

Reply:
Yes.